packages/driver/src/cy/assertions.coffee

```diff
@@ -372,12 +407,21 @@ create = (state, queue, retryFn) ->
 
       consoleProps: =>
         obj = {Command: "assert"}
+        parsedValues = parseValueActualAndExpected(value, actual, expected)
 
-        _.extend obj, parseValueActualAndExpected(value, actual, expected)
-
-        _.extend obj,
+        _.extend obj, parsedValues
+        
+        
+        _.extend obj, {
           Message: message.replace(bTagOpen, "").replace(bTagClosed, "")
```

Comment:
I would make a utility function here, also, will this replace just a single opening and closing tag or is this global replacement?

Reply:
Hmm, regex is global yes. But this PR did not touch that code